manageiq-operator/internal/controller/manageiq_controller.go

```diff
@@ -522,67 +526,60 @@ func (r *ManageIQReconciler) generatePostgresqlResources(cr *miqv1alpha1.ManageI
 }
 
 func (r *ManageIQReconciler) generateKafkaResources(cr *miqv1alpha1.ManageIQ) error {
-	secret, mutateFunc := miqtool.ManageKafkaSecret(cr, r.Client, r.Scheme)
-	if result, err := controllerutil.CreateOrUpdate(context.TODO(), r.Client, secret, mutateFunc); err != nil {
-		return err
-	} else if result != controllerutil.OperationResultNone {
-		logger.Info("Secret has been reconciled", "component", "kafka", "result", result)
-	}
-
-	hostName := getSecretKeyValue(r.Client, cr.Namespace, cr.Spec.KafkaSecret, "hostname")
-	if hostName != "" {
-		logger.Info("External Kafka Messaging Service selected, skipping kafka and zookeeper service reconciliation", "hostname", hostName)
-		return nil
-	}
-
-	kafkaPVC, mutateFunc := miqtool.KafkaPVC(cr, r.Scheme)
-	if result, err := controllerutil.CreateOrUpdate(context.TODO(), r.Client, kafkaPVC, mutateFunc); err != nil {
-		return err
-	} else if result != controllerutil.OperationResultNone {
-		logger.Info("PVC has been reconciled", "component", "kafka", "result", result)
-	}
+	if miqutilsv1alpha1.FindCatalogSourceByName(r.Client, "openshift-marketplace", "community-operators") != nil {
```

Comment:
Check to see if Openshift is being used to install Strimzi, by looking for the relevant catalogsource in the `openshift-marketplace` namespace